sign/utils.js

```diff
@@ -0,0 +1,15 @@
+//모듈화 예정입니다...
+
+const EMAIL_REGEX = /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
+
+export function emailCheck(email_address){     
```

Comment:
파라미터의 네이밍을 camelCase로 변경하면 좋을것 같습니다!
```suggestion
export function emailCheck(emailAddress){     
```